src/css/custom.css

```diff
@@ -18,11 +18,17 @@ body a:hover {
   width: 0;
 }
 
+/* Container for the /papers page */
+.archiveContainer {
+  margin: 2em;
+}
+
 /* Code Snippet  */
 .snippet-container {
-  --snippet-background: rgb(246, 248, 250);
+  --snippet-background: #F7FAFC;
```

Comment:
Matching styleguide

Reply:
I think the color came from a theme. It may not match the style guide, but the actual details of the snippet colors don't exactly match because they come from the theme. So, it seems weird that we would have the background match the color but not the inner colors. Can you include a screenshot of what it looks like now with this change?